default-grimoirelab-settings/setup.cfg

```diff
@@ -57,6 +57,11 @@ panels = true
 #raw_index = confluence_demo_raw
 #enriched_index = confluence_demo_enriched
 
+#[discourse]
+#raw_index = discourse_raw
+#enriched_index = discourse_enriched
+#no-archive = true
```

Comment:
Could you follow the same naming format?
- discourse_demo_raw
- discourse_demo_enriched

Reply:
Ah yes, that would make sense. Please see my latest commit for these updates